iothub/device/src/Common/Fx.cs

```diff
@@ -41,22 +32,6 @@ public static ExceptionTrace Exception
             }
         }
 
-        // Do not call the parameter "message" or else FxCop thinks it should be localized.
-        [Conditional("DEBUG")]
-        public static void Assert(bool condition, string description)
```

Comment:
Silly methods like this (and next) were unnecessary abstractions over a well-known built-in library.